shared_governance_base/models/mandate.py

```diff
@@ -1,11 +1,10 @@
-# -*- coding: utf-8 -*-
 from odoo import api, fields, models
 
 
 class Mandate(models.Model):
     _name = "sg.mandate"
 
-    _inherit = ["mail.thread", "ir.needaction_mixin"]
+    _inherit = ["mail.thread", "mail.activity.mixin"]
```

Comment:
Does this _just work_, or do I need to add some migratory stuff somewhere?

Reply:
from what i understand by reading the comment [here](https://github.com/OCA/OCB/blob/10.0/odoo/addons/base/ir/ir_needaction.py), only inheriting from `ir.needaction_mixin` without defining the corresponding methods has no effect, so we might as well just remove it. what are the reasons to add `mail.activity.mixin`?

Reply:
> what are the reasons to add mail.activity.mixin?

From what little research I did, `mail.activity.mixin` is the designated successor of `ir.needaction_mixin`. But if it's superfluous all the same, I can remove it.

Reply:
i think we need input from an analyst on this then.

Reply:
The history of module: we did a hackaton during a _mise au vert_ retreat. We probably had in mind to add some actions to be followed in the mandates but did not go all the way to implement them.
=> remove it